changelog.rst

```diff
@@ -11,6 +11,7 @@ Features:
 Bug fixes:
 ----------
 
+* Minor typo fixes in `pgclirc`. (Thanks: `anthonydb`_)
```

Comment:
You need to add your name at the end of this file, otherwise, there will be a rst syntax error which will fail the CI.

Reply:
Name added; looks like tests are passing now!